app/indexers/concerns/hyrax/indexes_basic_metadata.rb

```diff
@@ -1,9 +1,10 @@
 module Hyrax
-  # This module can be mixed in on an indexer in order to index the basic metadata fields
   module IndexesBasicMetadata
-    # We're overriding a method from ActiveFedora::IndexingService
-    def rdf_service
-      BasicMetadataIndexer
-    end
+    extend Deprecation
+
+    Deprecation.warn(
```

Comment:
Can you put this in an `included` block?  Otherwise it'll trigger when the code is eager loaded in production mode.

Reply:
Oh, sure.